code/_onclick/hud/ai_hud.dm

```diff
@@ -16,7 +16,7 @@
 
 /obj/screen/ai/camera_list/Click()
 	var/mob/living/silicon/ai/AI = usr
-	var/camera = input(AI, "Choose which camera you want to view", "Cameras") as null|anything in AI.get_camera_list()
+	var/camera = tgui_input_list(AI, "Choose which camera you want to view", "Cameras", AI.get_camera_list())
```

Comment:
Is it such a big problem that it is necessary to prohibit opening multiple identical windows?
I think it may be convenient for the AI on the one hand, as there are camera blanks to jump to, but on the other hand these windows take up screen space in any case.
If so, I think it's best to delete in this particular case

Reply:
I don't really see an issue with it, we can go back to tweak if needed.